github_activity/cli.py

```diff
@@ -109,6 +110,12 @@
     default=None,
     help=("""The branch or reference name to filter pull requests by"""),
 )
+parser.add_argument(
+    "--all",
+    default=False,
+    action="store_true",
+    help=("""Whether to include all the GitHub tags"""),
```

Comment:
I think we should explain a bit more what this means, as I don't think people will realize this means "generate a gigantic markdown file for every tag that exists in the repository". How about this
```suggestion
    help=("""Include all GitHub tags in the generated Changelog. Useful when generating a changelog for the entire history of a repository."""),
```